docs/topics/settings.rst

```diff
@@ -229,8 +229,7 @@ BOT_NAME
 Default: ``'scrapybot'``
 
 The name of the bot implemented by this Scrapy project (also known as the
-project name). This will be used to construct the User-Agent by default, and
-also for logging.
+project name) and also for logging.
```

Comment:
So it now reads "The name of the bot implemented by this Scrapy project (also known as the project name) and also for logging."

Reply:
```The name of the bot implemented by this Scrapy project (also known as the project name). This name will be used for the logging too.```
is it need to rewrite like this?

Reply:
This looks better.